locales/en/plugin__kubevirt-plugin.json

```diff
@@ -350,7 +350,7 @@
   "Details": "Details",
   "Detected file extension is {{fileNameExtension}}": "Detected file extension is {{fileNameExtension}}",
   "Developer preview": "Developer preview",
-  "Developer preview features are not intended to be used in production environments. The clusters deployed with the developer preview features are considered to be development clusters and are not supported through the Red Hat Customer Portal case management system.": "Developer preview features are not intended to be used in production environments. The clusters deployed with the developer preview features are considered to be development clusters and are not supported through the Red Hat Customer Portal case management system.",
+  "Developer preview features are not intended to use in production environments. The clusters deployed with the developer preview features are developmental clusters and are not currently supported by Red Hat.": "Developer preview features are not intended to use in production environments. The clusters deployed with the developer preview features are developmental clusters and are not currently supported by Red Hat.",
```

Comment:
I would like this reviewed by docs because it is verbose. I can't tag Sabrina Jess, however.

Reply:
Copy and send her the link for these comments.

Reply:
Developer preview features are not currently supported by Red Hat and not intended for use in production environments.